uv.lock

```diff
@@ -226,23 +226,24 @@ filecache = [
 
 [[package]]
 name = "canonical-sphinx"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
You beat me to the punch on this one. Any chance you could incorporate the appropriate [config changes](https://github.com/canonical/canonical-sphinx/pull/74) to enable the edit button?


> * If `html_theme_options.source_edit_link` is set in conf.py and points to a hosted Git repo, the edit button is enabled. The button uses the link in the `github_folder` (old) and `repo_folder` (new) settings.
> ...
> * Rename `github_folder` config setting to `repo_folder`.
> * Rename `github_version` config setting to `repo_default_branch`.

Reply:
Sure thing. If I'm reading this right, the only thing that was missing was setting `html_theme_options.source_edit_link`.

A caveat here is that the button works fine for pages that are part of rockcraft's docs (e.g. the main page), but cannot work for pages that come from craft-parts (it just 404's)

Reply:
> cannot work for pages that come from craft-parts

Understood. It's an unfortunate problem that we can't really design around because it's a side effect of how we're bringing in content. Other than hiding the button and handling all feedback about it, I don't know of a good UX for pages that are completely transcluded.

Reply:
> If I'm reading this right, the only thing that was missing was setting html_theme_options.source_edit_link.

We'll need to change `github_folder` and `github_version`, too. The build will provide a nag message otherwise.

Reply:
Those fields were already absent in `conf.py`, am I missing something? It builds cleanly, and the edit button [seems to be working fine](https://canonical-ubuntu-documentation-library--838.com.readthedocs.build/rockcraft/en/838/)

Reply:
My mistake, was looking at the wrong keys. Looks good!